lib/rworkflow/lifecycle.rb

```diff
@@ -7,9 +7,8 @@ class Lifecycle
 
     DEFAULT_STATE_OPTIONS = {
       cardinality: State::DEFAULT_CARDINALITY,
-      priority: State::DEFAULT_PRIORITY,
       policy: State::STATE_POLICY_NO_WAIT
-    }
+    }.freeze
 
     def initialize(state_class: State, state_options: {})
       @state_options = DEFAULT_STATE_OPTIONS.merge(state_options)
```

Comment:
should really be self.class::DEFAULT_STATE_OPTIONS to allow subclasses to overload the constant